PixelMatcher/Views/MainWindow.xaml

```diff
@@ -1,75 +1,137 @@
+                    </Image.RenderTransform>
+                </Image>
+                <i:Interaction.Triggers>
+                    <i:EventTrigger EventName="MouseDown">
+                        <i:InvokeCommandAction Command="{Binding ImageMouseDownCommand}" PassEventArgsToCommand="True" />
+                    </i:EventTrigger>
+                    <i:EventTrigger EventName="MouseMove">
+                        <i:InvokeCommandAction Command="{Binding ImageMouseMoveCommand}" PassEventArgsToCommand="True" />
+                    </i:EventTrigger>
+                    <i:EventTrigger EventName="MouseUp">
+                        <i:InvokeCommandAction Command="{Binding ImageMouseUpCommand}" PassEventArgsToCommand="True" />
+                    </i:EventTrigger>
+                </i:Interaction.Triggers>
+            </Canvas>
         </Border>
-        <Border Height="32" VerticalAlignment="Top">
+        <Border Height="32" VerticalAlignment="Top" Panel.ZIndex="100500">
             <DockPanel VerticalAlignment="Center">
-                <StackPanel Orientation="Horizontal" VerticalAlignment="Center" Margin="5,0,0,0">
+                <StackPanel Margin="5,0,0,0" VerticalAlignment="Center" Orientation="Horizontal">
                     <TextBlock Style="{StaticResource ImagesText}">
-                        <Run Text="Image: "/>
-                        <Run Text="{Binding ImagePosition, FallbackValue=0}"/>
-                        <Run Text="/"/>
-                        <Run Text="{Binding Images.Count, FallbackValue=0, Mode=OneWay}"/>
+                        <Run Text="Image: " />
+                        <Run Text="{Binding ImageIndex, FallbackValue=0}" />
+                        <Run Text="/" />
+                        <Run Text="{Binding Images.Count, FallbackValue=0, Mode=OneWay}" />
                     </TextBlock>
                     <TextBlock Style="{StaticResource NoImagesText}">
-                        <Run Text="Drop or paste image."/>
+                        <Run Text="Drop or paste image." />
                     </TextBlock>
-                    <TextBlock Text="Opacity:" Margin="10,0,5,0"/>
-                    <TextBlock Text="{Binding ImageOpacity, StringFormat={}{0:0.##}}"/>
+                    <TextBlock Margin="10,0,5,0" Text="Opacity:" />
+                    <TextBlock Text="{Binding ImageOpacity, StringFormat={}{0:0.##}}" />
+                    <Slider Width="80"
+                            Margin="0,0,5,0"
+                            LargeChange="0.33"
+                            Maximum="{x:Static viewModels:MainViewModel.MaximumOpacity}"
+                            Minimum="{x:Static viewModels:MainViewModel.MinimumOpacity}"
+                            SmallChange="0.1"
+                            Value="{Binding ImageOpacity}" />
+                    <TextBlock Margin="10,0,5,0" Text="Zoom:" />
+                    <TextBlock Margin="0,0,5,0" Text="{Binding ZoomLevel, StringFormat=x{0:0.0}}" />
```

Comment:
Please, add fixed size to this element. Changing image zoom can cause all elements to the right to tremble.